showcase/config/targets.js

```diff
@@ -6,9 +6,9 @@
 'use strict';
 
 const browsers = [
-  'last 2 Chrome versions',
-  'last 2 Firefox versions',
-  'last 2 Safari versions',
+  'last 1 Chrome versions',
+  'last 1 Firefox versions',
+  'last 1 Safari versions',
```

Comment:
last 2 versions is still our current support

Reply:
I can revert that if this is HDS stance or browser support, but this is what came from blueprint and I think there is an RFC to back that up, but you tell me